packages/components/dialog/src/token.ts

```diff
@@ -0,0 +1,14 @@
+import type { ComputedRef, CSSProperties, InjectionKey, Ref } from 'vue'
+import type { useNamespace } from '@element-plus/hooks'
+
+export type DialogContext = {
+  dialogRef: Ref<HTMLElement | null>
+  headerRef: Ref<HTMLElement | null>
```

Comment:
inference component typing

Reply:
I'm not following, would you mind elaborate a little bit?